.github/scripts/setup/mac-sign.sh

```
@@ -0,0 +1,107 @@
+  echo -e "  --macos-skip-root-certificate\t\tSkip importing Apple Root certificate. Useful when running in already configured environment."
+  echo -e "  --help\t\t\t\tShow this help text and exit."
+  echo
+  echo "Examples:"
+  echo "  $0 sign.hcl"
+}
+
+function main {
+  local mac_skip_root_certificate=""
+  local assets=()
+
+  while [[ $# > 0 ]]; do
```

Shellcheck (0.10.0)

[error] > is for string comparisons. Use -gt instead.

(SC2071)